Source/missiles.h

```diff
@@ -138,7 +143,34 @@ Direction16 GetDirection16(Point p1, Point p2);
 void DeleteMissile(int i);
 bool MonsterTrapHit(int m, int mindam, int maxdam, int dist, missile_id t, bool shift);
 bool PlayerMHit(int pnum, Monster *monster, int dist, int mind, int maxd, missile_id mtype, bool shift, int earflag, bool *blocked);
+
+/**
+ * @brief Sets the missile sprite to the given sheet frame
+ * @param missile this object
+ * @param dir Sprite frame, typically representing a direction but there are some exceptions (arrows being 1 indexed, directionless spells)
+*/
 void SetMissDir(Missile &missile, int dir);
+
+/**
+ * @brief Sets the sprite for this missile so it matches the given Direction
+ * @param missile this object
+ * @param dir Desired facing
+*/
+inline void SetMissDir(Missile &missile, Direction dir)
+{
+	SetMissDir(missile, static_cast<int>(dir));
+}
+
+/**
+ * @brief Sets the sprite for this missile so it matches the given Direction16
+ * @param missile this object
+ * @param dir Desired facing as a hellfire direction

```

Comment:
Hellfire did not add this, it did add new flare types with a directional sprite which might be where this assumption comes from.
```suggestion
 * @param dir Desired facing at a 22.8125 degree resolution
```